swift/ql/test/extractor-tests/generated/expr/SingleValueStmtExpr/CONSISTENCY/CfgConsistency.expected

```diff
@@ -0,0 +1,2 @@
+deadEnd
+| test.swift:1:8:1:13 | x |
```

Comment:
I think this means that the problem with the new dead ends is just that the new extracted AST node needs to be added to control flow as well. This can happen in a follow up PR though :+1:

Reply:
It's indeed in the followup PR https://github.com/github/codeql-c-team/issues/2042